pkg/sdk/poc/generator/templates/sub_templates/implementation_functions.tmpl

```diff
@@ -13,9 +14,17 @@
         return resultList, nil
         }
     {{ else if eq .Name "ShowByID" }}
+        {{ $idkind := .ObjectInterface.ObjectIdentifierKind }}
         func (v *{{ $impl }}) ShowByID(ctx context.Context, id {{ .ObjectInterface.IdentifierKind }}) (*{{ .ObjectInterface.NameSingular }}, error) {
-        // TODO: adjust request if e.g. LIKE is supported for the resource
-        {{ $impl }}, err := v.Show(ctx, NewShow{{ .ObjectInterface.NameSingular }}Request())
+            request := NewShow{{ .ObjectInterface.NameSingular }}Request()
+            {{- if not .ShowByIDFiltering }}
```

Comment:
is it a desired situation that no filtering is provided? I think we should enforce putting option here, e.g. some special option for no filtering or smth like that. We can discuss it.

Reply:
agreed, we could add `ShowByIDNoFiltering` option